python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -719,12 +719,24 @@ def _process_auto_materialize_tick_generator(
 
             if sensor:
                 selection = check.not_none(sensor.asset_selection)
+                repository_origin = check.not_none(repository).get_remote_origin()
                 # resolve the selection against just the assets in the sensor's repository
                 repo_asset_graph = check.not_none(repository).asset_graph
-                eligible_keys = selection.resolve(repo_asset_graph) | selection.resolve_checks(
+                resolved_keys = selection.resolve(repo_asset_graph) | selection.resolve_checks(
                     repo_asset_graph
                 )
                 eligibility_graph = repo_asset_graph
+
+                # Ensure that if there are two asset keys defined in the same code location with automation
```

Comment:
this comment should be "same deployment" right?

Reply:
yeah updated comment, my bad